src/test/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/CommandTestUtil.java

```diff
@@ -116,6 +117,16 @@ public static void assertCommandSuccess(Command command, Model actualModel, Comm
      * Convenience wrapper to {@link #assertCommandSuccess(Command, Model, CommandResult, Model)}
      * that takes a string {@code expectedMessage}.
      */
+    public static void assertCommandSuccess(Command command, Model actualModel, String expectedMessage,
+                                            Model expectedModel) {
```

Comment:
```suggestion
            Model expectedModel) {
```
This should be the more appropriate indentation level, according to the standard. Ignore the indentation IntelliJ gives you.